testnet/stacks-node/src/tests/mod.rs

```diff
@@ -464,14 +464,17 @@ fn make_microblock(
 pub fn select_transactions_where(
     blocks: &Vec<serde_json::Value>,
     test_fn: fn(&StacksTransaction) -> bool,
+    epoch_id: StacksEpochId,
 ) -> Vec<StacksTransaction> {
     let mut result = vec![];
     for block in blocks {
         let transactions = block.get("transactions").unwrap().as_array().unwrap();
         for tx in transactions.iter() {
             let raw_tx = tx.get("raw_tx").unwrap().as_str().unwrap();
             let tx_bytes = hex_bytes(&raw_tx[2..]).unwrap();
-            let parsed = StacksTransaction::consensus_deserialize(&mut &tx_bytes[..]).unwrap();
+            let parsed =
```

Comment:
This change shouldn't be necessary since these blocks are already mined and processed